Vagrantfile

```diff
@@ -1,27 +1,30 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
-# Provision root account with the insecure Vagrant ssh key.
```

Comment:
Vagrant removes its insecure key now -- the ansible config puts in the user's key, not any vagrant-controlled ones.